### DISCUSSION TITLE
Add sample content to Pico Block theme

### DISCUSSION THREAD

**Discussion by stratofax** [cadentcom/pico-wp-block-theme#26] [Ideas]
To show the theme in use, we need to add sample content.

**Comment by stratofax** [comment]
@ratulkhan74 what's the best way for us to add sample content to pico?

**Reply by stratofax** [reply]
For specific discussion of the home page sample content, please see #27

**Reply by ratulkhan74** [reply]
@stratofax please check #27

**Reply by stratofax** [reply]
Let's leave this discussion open in case we want to add content that's not on the home page (a sample post or page, for example)
